assembl/static2/css/components/fictionSideCommentBox.scss

```diff
@@ -0,0 +1,66 @@
+.side-comment-box {
+  position: absolute;
+  z-index: 200;
+
+  .theme-box {
+    border-left: none;
+
+    .extracts-nb-msg {
+      padding: 10px 20px;
+    }
+
+    .harvesting-box-header {
+      border: 0;
+    }
+
+    .harvesting-box-body {
+      padding: 10px 5px 10px;
+    }
+
+    .body-container {
+      .previous-extract,
+      .next-extract {
+        flex-basis: 10px;
+        padding: 0;
+        font-size: 8px;
+      }
+
+      .next-extract {
+        margin-top: 10px;
+      }
+
+      .extract-body {
+        min-width: 255px;
+        margin: 0 5px 10px;
+      }
+    }
+
+    .submit-comment {
+      padding: 0 15px;
+    }
+  }
+
+  .arrow-left {
+    position: absolute;
```

Comment:
I see quite a few ```position: absolute;``` here, is this all responsive 😛 ?

Reply:
Nope, its a feature only for desktop